packages/lodestar-cli/test/e2e/constants.ts

```diff
@@ -0,0 +1,2 @@
+export const rootDir = ".tmp";
```

Comment:
Maybe it would be better to use tmp directory dependency just to make sure we don't collide with some directory someone was using and delete him that

Reply:
sorry, just saw this after i merged.  do you mean that we should rename it to something like tmpLodestarE2E or something like that?

Reply:
There's a library called `tmp` which can be used, it's used in `lodestar/test`

Reply:
ahh ok.  i'll start whipping up a PR for that here in a sec